### ISSUE TITLE
RCS1160 False positive on 'protected internal' usage

### ISSUE THREAD

**Issue by legionaryu** [JosefPihrt/Roslynator#842]
**Product and Version Used**: VS Code Extension v3.2.2 (josefpihrt-vscode.roslynator)

**Steps to Reproduce**:
Declare an internal abstract class and make its constructor 'protected internal'
```csharp
internal abstract class DummyClass
{
    protected internal DummyClass() { }
}
```

**Actual Behavior**:
The rule RCS1160 falsely claims that the constructor is publicly available.
```csharp
internal abstract class DummyClass
{
    protected internal DummyClass() { }
                                ^^^^^^^^^  => Abstract type should not have public constructors. RCS1160
}
```

**Expected Behavior**:
The rule RCS1160 should accept the combination 'protected internal' as non-public.
```csharp
internal abstract class DummyClass
{
    protected internal DummyClass() { }
}
```
